gui/mozregui/log_report.py

```diff
@@ -1,7 +1,14 @@
 from PyQt4.QtCore import QObject, pyqtSlot as Slot, pyqtSignal as Signal
-from PyQt4.QtGui import QPlainTextEdit, QTextCursor
+from PyQt4.QtGui import QPlainTextEdit, QTextCursor, QColor, \
+    QTextCharFormat
 from datetime import datetime
 
+color = {
```

Comment:
Please write this as COLORS (uppercase by convention, as it is a global variable)